Common Example code/PaymentsController.swift

```diff
@@ -185,6 +185,7 @@ internal final class PaymentsController {
             }
         case let .failure(error):
             currentComponent?.stopLoading(withSuccess: false) { [weak self] in
+                self?.presenter?.dismiss(completion: nil)
```

Comment:
Does it show the alert if you try ApplePay on v66? 🤔

Reply:
yes, it does.